bin/rqt_graph

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+
+import sys
+
+from rqt_graph.main import main
+
+sys.exit(main())
```

Comment:
Since the file isn't installed to `bin` it should probably be in differently named directory, e.g. `scripts`.

Reply:
Makes sense, see 44266fc

We should consider making a similar change in other rqt packages that use a directory named `bin` (e.g. [rqt_gui](https://github.com/ros-visualization/rqt/tree/crystal-devel/rqt_gui)).